pkg/apis/cr/v1alpha1/types.go

```diff
@@ -265,13 +267,15 @@ type CredentialType string
 const (
 	CredentialTypeKeyPair CredentialType = "keyPair"
 	CredentialTypeSecret  CredentialType = "secret"
+	CredentialTypeKopia   CredentialType = "kopia"
 )
 
 // Credential
 type Credential struct {
-	Type    CredentialType   `json:"type"`
-	KeyPair *KeyPair         `json:"keyPair,omitempty"`
-	Secret  *ObjectReference `json:"secret,omitempty"`
+	Type         CredentialType   `json:"type"`
+	KeyPair      *KeyPair         `json:"keyPair,omitempty"`
+	Secret       *ObjectReference `json:"secret,omitempty"`
+	KopiaSecrets *KopiaSecret     `json:"kopiaSecrets,omitempty"`
```

Comment:
```suggestion
	KopiaServerSecret *KopiaServerSecret     `json:"kopiaSecrets,omitempty"`
```
Can we make this and the `param.Profile` consistent?